decentraland/unity-renderer
# chore: chat&friends controller refactor into bridges

## What does this PR change?

Separated the responsibilities of the chat & friend controllers into bridges.
Bridges handles the communication with kernel. Controllers handles DTOs, process and them emit the information to the application domain.
It will also ease the migration of chat to protobuf.

## How to test the changes?

1. Go to: https://play.decentraland.org/?renderer-branch=chore/social-bridges-refactor
2. Check that channels are loaded correctly, in the conversation list and channel browser
3. Try joining, leaving, sending and receiving messages in channels
4. Open direct messages, check history, receive and send messages
5. Open #nearby, receive and send messages
6. Open the friends window
7. Send, receive, add, remove friends/requests

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
